tests/test_reduce_sum.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+import numpy as np
+import tensorflow as tf
+import tensorflow_encrypted as tfe
+
+
+class Testconcat(unittest.TestCase):
```

Comment:
From #183, we should rename this

Reply:
Good catch, I hate when people don't change it ... Now I'm hating myself 🥇